static/app/views/replays/detail/layout/focusArea.tsx

```diff
@@ -58,13 +66,28 @@ function FocusArea({}: Props) {
         />
       );
     case TabKey.CONSOLE:
-    default:
+    default: {
+      const hasErrorTab = organization.features.includes('session-replay-errors-tab');
+
+      if (hasErrorTab) {
+        return (
+          <Console
+            breadcrumbs={replay?.getConsoleCrumbs()}
+            startTimestampMs={replay?.getReplay()?.started_at?.getTime() || 0}
+          />
+        );
+      }
+
+      const breadcrumbs = !replay
+        ? undefined
+        : [...replay.getConsoleCrumbs(), ...replay.getIssueCrumbs()];
```

Comment:
Hmm what about

```suggestion
      const breadcrumbs = !replay
        ? undefined
        : hasErrorTab
        ? replay.getConsoleCrumbs()
        : [...replay.getConsoleCrumbs(), ...replay.getIssueCrumbs()];
```

Reply:
I had that to start with but changed it to be like this so that it's easier to highlight a few lines and delete them later when the feature flag gets cleaned up. 

But the negative ternary condition looks dumb in either case. We don't need that.